Scripts/RemediationScripts/Remediate-ConfigureAADAuthExtOnLinuxVMSS.ps1

```diff
@@ -0,0 +1,1109 @@
+<###
```

Comment:
Can this BRS supports auto remediation from UI, if yes, please add that config as well?

Reply:
This BRS doesn't supports auto remediation as after installing the function with Add method it takes some time for the extension to be in succeeded provisioning state and for that we have validate function which checks he provisioning state.